### PR TITLE
feat: implement firebase auth script

### PR DESCRIPTION
**Pull request by anversa-pro**
## Describe your changes
Add constants of auth config to environment and create the script to get a token for tester user
## Issue ticket number and link
https://trello.com/c/xxXo6U7i
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

File: environment.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
+const dotenv = require('dotenv')
+
 const DEFAULT_DATABASE = 'mongodb'
 const EVENTS_COLLECTION_NAME = 'events'
 const HEADQUARTERS_COLLECTION_NAME = 'headquarters'
+const USER_ID = 'testuser@chupito.com'
+const USER_PWD = 'TesT#975'
```
Comment (jlora-mck) [inline environment.js:7]: Please remove

File: providers/firebase-auth.application.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+account.type = 'service_account'
+account.project_id =  'chupito-c5p5t'
+account.private_key_id = firebaseConfig.AUTH_PRIVATE_KEY_ID
+account.private_key = firebaseConfig.AUTH_PRIVATE_KEY
+account.client_email = firebaseConfig.AUTH_CLIENT_EMAIL
+account.client_id = firebaseConfig.AUTH_CLIENT_ID
+account.auth_uri = 'https://accounts.google.com/o/oauth2/auth'
+account.token_uri = 'https://oauth2.googleapis.com/token'
+account.auth_provider_x509_cert_url= 'https://www.googleapis.com/oauth2/v1/certs'
+account.client_x509_cert_url = firebaseConfig.AUTH_CLIENT_X509_CERT_URL
```
Comment (jlora-mck) [inline providers/firebase-auth.application.js:23]:
```suggestion
dotenv.config()

const account = { 
...serviceAccount 
type: 'service-account',
.... // repeat for all the others properties
}
```

File: providers/firebase-auth.application.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+let app = null
+
+module.exports = () => { 
+  if (!app) {
+    app = admin.initializeApp({
+      credential: admin.credential.cert(account),
+    })
+  }
+
+  return app
```
Comment (jlora-mck) [inline providers/firebase-auth.application.js:34]: Not sure why do we need this `if (!app)`, is this code running multiple times ?

File: environment.js (hunk 1 above)
Comment (anversa-pro) [reply environment.js]: Removed

File: providers/firebase-auth.application.js (hunk 3 above)
Comment (anversa-pro) [reply providers/firebase-auth.application.js]: File removed, no longer in use.

File: providers/firebase-auth.application.js (hunk 2 above)
Comment (anversa-pro) [reply providers/firebase-auth.application.js]: File removed, no longer in use

File: scripts/firebase-auth-test.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  apiKey: process.env.AUTH_API_KEY,
+  authDomain: process.env.AUTH_DOMAIN,
+  projectId: process.env.AUTH_PROJECT_ID,
+  storageBucket: process.env.AUTH_STORAGE_BUCKET,
+  messagingSenderId: process.env.AUTH_MESSAGING_SENDER_ID,
+  appId: process.env.AUTH_APP_ID,
+  measurementId: process.env.AUTH_MEASUREMENT_ID,
+}
+
+let app = null
```
Comment (FelipeLosada1996) [inline scripts/firebase-auth-test.js:17]: Is not necessary to give then a value by default.

**Review by FelipeLosada1996 — APPROVED** [review]

**Review by jose-sal-y-rosas — APPROVED** [review]
👍🏼

**Review by jlora-mck — APPROVED** [review]

**Review by FelipeLosada1996 — APPROVED** [review]

**Pull request merged**